prml/generic-asset/src/lib.rs

```diff
@@ -484,6 +499,8 @@ decl_event! {
 		Minted(AssetId, AccountId, Balance),
 		/// Asset burned (asset_id, account, amount).
 		Burned(AssetId, AccountId, Balance),
+		/// Asset balance storage has been reclaimed due to falling below the existential deposit
+		Purged(AssetId, AccountId, Balance),
```

Comment:
something like this will be nicer for dapps
```suggestion
		DustReclaimed(AssetId, AccountId, Balance),
```